kratos/includes/variables.h

```diff
@@ -186,6 +186,7 @@ namespace Kratos
     KRATOS_DEFINE_VARIABLE(double, NORMAL_SENSITIVITY )
     KRATOS_DEFINE_VARIABLE(double, NUMBER_OF_NEIGHBOUR_ELEMENTS )
     KRATOS_DEFINE_VARIABLE(bool, UPDATE_SENSITIVITIES )
+    KRATOS_DEFINE_VARIABLE(double, NUMERICAL_DIFFUSION )
```

Comment:
Why do you define this in the core and not in the App you are using it?

Reply:
Since recently it is possible to define the same variable in different apps
In this PR you only use it in AdjointFluid, so I didn't know you want to use it in Trilinos

With this one I am not sure though, maybe it would make sense to add it to the core, what do the others think?

Reply:
Now that the variables can be redefined in multiple applications, we could try moving it back to the application.

Reply:
moved all the variables to AdjointFluidApplication.